src/chain.ts

```diff
@@ -27,4 +27,4 @@ export const isArbitrum = (chainId: ChainId): boolean => {
   return chainId === 42161;
 };
 
-export const allSupportedChains = Object.keys(Chains).map(key => Number(key));
+export const allSupportedChains = Object.keys(Chains).map((key) => Number(key));
```

Comment:
These parentheses are also recommended, were missing in the old lint config (from tsdx)